tests/cache/tests.py

```diff
@@ -2489,6 +2488,21 @@ def test_304_response_has_http_caching_headers_but_not_cached(self):
         self.assertIn('Cache-Control', response)
         self.assertIn('Expires', response)
 
+    def test_per_thread(self):
+        """The cache instance is different for each thread."""
```

Comment:
I know this is just a test, but to increase future readability I would suggest using talking names and not single characters (e.g. `c`, `t`, `x`) to store variables, and I would also remove unnecessary blank lines

Reply:
I updated variable names.

> and I would also remove unnecessary blank lines

I don't see any unnecessary blank lines :thinking:

Reply:
I'm sorry, the short variables had misled me, in fact there aren't any. Thanks

Reply:
I copied that from a similar test, but I agree it is not a good reason to continue with a bad practice. Thanks for the corrections!